src/main/java/de/tum/in/test/api/structural/MethodTestProvider.java

```diff
@@ -104,12 +104,14 @@ protected static void checkMethods(String expectedClassName, Class<?> observedCl
 			var expectedModifiers = getExpectedJsonProperty(expectedMethod, JSON_PROPERTY_MODIFIERS);
 			var expectedAnnotations = getExpectedJsonProperty(expectedMethod, JSON_PROPERTY_ANNOTATIONS);
 			var expectedReturnType = expectedMethod.getString(JSON_PROPERTY_RETURN_TYPE);
+			boolean strictParameterOrder = getExpectedJsonBooleanProperty(expectedMethod, JSON_PROPERTY_Strict_Order);
```

Comment:
```suggestion
			var strictParameterOrder = getExpectedJsonBooleanProperty(expectedMethod, JSON_PROPERTY_Strict_Order);
```
Would `var` make more sense in this context (everything else is `var`)?